src/components/Identity/createComponent.js

```diff
@@ -36,22 +36,23 @@ export default (processIdSyncs, config, logger, optIn, eventManager) => {
           const identityQuery = {
             fetch: [ecidNamespace]
           };
+          const configOverridesMeta = {};
 
-          // TODO: Are these things being moved to the Konductor/config service?
           if (config.idSyncEnabled) {
-            identityQuery.exchange = true;
-            if (config.idSyncContainerId !== undefined) {
-              identityQuery.containerId = config.idSyncContainerId;
+            configOverridesMeta.idSync = {
+              enabled: true
+            };
+            if (config.containerId !== undefined) {
+              configOverridesMeta.idSync.containerId = 40;
             }
           }
 
-          if (!config.thirdPartyCookiesEnabled) {
-            identityQuery.thirdPartyCookiesEnabled = false;
```

Comment:
Catalin says this went away entirely. I don't yet know what it was doing entirely or what we might need to do to compensate. It might be related to https://github.com/adobe/alloy/pull/318/files#r359474466